Userland/Libraries/LibWeb/Internals/Internals.h

```diff
@@ -29,6 +29,8 @@ class Internals final : public Bindings::PlatformObject {
     void send_key(HTML::HTMLElement&, String const&, WebIDL::UnsignedShort modifiers);
     void commit_text();
 
+    void send_key_right_keydown_event(WebIDL::UnsignedShort modifiers);
```

Comment:
Does `send_key` not work for this? I added it for non-textual keys like this

Reply:
indeed, it does. I was a bit confused by this function having `HTML::HTMLElement&` as the first argument.